pom.xml

```diff
@@ -26,6 +26,9 @@
     <changelist>999999-SNAPSHOT</changelist>
     <gitHubRepo>jenkinsci/${project.artifactId}-plugin</gitHubRepo>
     <jenkins.version>2.303.3</jenkins.version>
+    <spotbugs.effort>Max</spotbugs.effort>
+    <spotbugs.failOnError>true</spotbugs.failOnError>
```

Comment:
Fail on error is the configured default from the parent pom.  No need to configure it here.
```suggestion
```